python_examples/luxembourg_example/luxembourg_repeat_sims_loc_cst.py

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
This file looks nearly identical to `python_examples/luxembourg_example/luxembourg_repeat_sims_loc_change.py` - please combine them and just describe the necessary modifications in the comments to run one or the other

Reply:
Changed